jsapp/js/account/plan.component.tsx

```diff
@@ -0,0 +1,442 @@
+      .finally(() => setButtonDisabled(!buttonsDisabled));
+  };
+
+  const managePlan = () => {
+    if (!state.organization?.uid || buttonsDisabled) {
+      return;
+    }
+    setButtonDisabled(buttonsDisabled);
+    postCustomerPortal(state.organization?.uid)
+      .then((data) => {
+        if (!data.url) {
+          alert('There has been an issue, please try again later.');
+        } else window.location.assign(data.url);
+      })
+      .finally(() => setButtonDisabled(!buttonsDisabled));
+  };
+
+  // Get feature items and matching icon boolean
+  const getListItem = (listType: string, plan: string) => {
+    const listItems: {icon: boolean; item: string}[] = [];
+    filterPrices().map((price) =>
+      Object.keys(price.metadata).map((featureItem: string) => {
+        const numberItem = featureItem.lastIndexOf('_');
+        const currentResult = featureItem.substring(numberItem + 1);
+
+        const currentIcon = `feature_${listType}_check_${currentResult}`;
+        if (
+          featureItem.includes(`feature_${listType}_`) &&
+          !featureItem.includes(`feature_${listType}_check`) &&
+          price.name === plan
+        ) {
+          const keyName = `feature_${listType}_${currentResult}`;
+          let iconBool = false;
+          const itemName: string = price.metadata[keyName];
+          if (price.metadata[currentIcon] !== undefined) {
+            iconBool = JSON.parse(price.metadata[currentIcon]);
+            listItems.push({icon: iconBool, item: itemName});
+          }
+        }
+      })
+    );
+    return listItems;
+  };
+
+  const checkMetaFeatures = () => {
+    if (state.products.length > 0) {
+      filterPrices().map((price) =>
+        Object.keys(price.metadata).map((featureItem: string) => {
+          if (
+            featureItem.includes(`feature_support_`) ||
+            featureItem.includes(`feature_advanced_`) ||
+            featureItem.includes(`feature_addon_`)
+          ) {
+            setShowExpand(true);
+          } else {
+            setShowExpand(false);
+          }
+        })
+      );
+    }
+  };
+  if (!state.products.length) return null;
+  return (
+    <div className={styles.accountPlan}>
+      <div className={styles.plansSection}>
+        <form className={styles.intervalToggle}>
+          <input
+            type='radio'
+            id='switch_left'
+            name='switchToggle'
+            value='year'
+            onChange={() => dispatch({type: 'year'})}
+            checked={!state.filterToggle}
+          />
+          <label htmlFor='switch_left'>{t('Annual')}</label>
+
+          <input
+            type='radio'
+            id='switch_right'
+            name='switchToggle'
+            value='month'
+            onChange={() => dispatch({type: 'month'})}
+            checked={state.filterToggle}
+          />
+          <label htmlFor='switch_right'> {t('Monthly')}</label>
+        </form>
+        <div
+          className={styles.currentPlan}
```

Comment:
I find it a bit confusing that it's not the part of the `.planContainer` that is subscribed. Would it be hard to change it now?

Reply:
I'm not sure I follow because then it would get rendered multiple times.

Reply:
I thought about it being put inside that `.map` loop, just with a conditional rendering:

```ts
{filterPrices().map((price: Price, i: number) => (
  {isSubscribedProduct(price) &&
    <div
      className={styles.currentPlan}
…
```

Reply:
This way you will get it only once and it would be easier to style it being positioned above proper price box. You'd propably need to add one more wrapper inside that loop

Reply:
As an argument for moving this - because it is being rendered separately, it's not synced properly for me now:
<img width="484" alt="Screenshot 2023-04-03 at 13 20 32" src="https://user-images.githubusercontent.com/2521888/229495033-f0977c11-9578-44fe-a7eb-1287e7df8fa2.png">

Reply:
I agree now and moved it